AppInspector/Commands/TagTestCommand.cs

```diff
@@ -23,7 +23,6 @@ enum TagTestType { RulesPresent, RulesNotPresent }
         private string _arg_srcPath;
         private string _arg_customRulesPath;
         private string _arg_outputFile;
-        private bool _arg_ignoreDefaultRules;
```

Comment:
Is there still value in supporting suppressing the default rules here? Should still be possible, just requires dumping out the embedded filters if the user requests it.

Reply:
no there is no value in testing against the default set only a custom set.  testing against the default set is equivalent to just running analyze command which is why I removed it.